router/profiles.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express')
+const controller = require('../controllers/profiles')
+
+const router = express.Router()
+
+const {
+  createProfile,
+  updateProfile,
+  deleteProfile,
+  listProfiles
+} = controller
+
+router.post('/', createProfile)
+router.put('/', updateProfile)
+router.delete('/', deleteProfile)
```

Comment:
you have to pass the id through the path in `updateProfile` and `deleteProfile` endpoints